patches/@nodelib__fs.walk@2.0.0.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/out/providers/stream.js b/out/providers/stream.js
+index 4e6cb5f089caf5c31563d37f471a39f3351af91d..2af8b5f346141c825711a6d9030178c787a5394e 100644
+--- a/out/providers/stream.js
++++ b/out/providers/stream.js
+@@ -26,10 +26,12 @@ class StreamProvider {
+         return new node_stream_1.Readable({
+             objectMode: true,
+             read: () => { },
+-            destroy: () => {
+-                if (!this.#reader.isDestroyed) {
+-                    this.#reader.destroy();
+-                }
++            destroy: (error, callback) => {
```

Comment:
It will not work unless this patch is applied to each repository where the tool is used.

Reply:
Oh, I forgot it. 😮  Let me find other workaround for this issue...